hpat/hiframes/series_kernels.py

```diff
@@ -528,7 +528,6 @@ def gt_f(a, b):
     'nsmallest_default': lambda A, name: hpat.hiframes.api.init_series(hpat.hiframes.api.nlargest(A, 5, False, lt_f), None, name),
     'head': lambda A, I, k, name: hpat.hiframes.api.init_series(A[:k], None, name),
     'head_index': lambda A, I, k, name: hpat.hiframes.api.init_series(A[:k], I[:k], name),
-    'median': lambda A: hpat.hiframes.api.median(A),
```

Comment:
do not delete this. Comment it out if you need. In this PR this code is needed for parallelism.